tt_metal/hw/firmware/src/erisc.cc

```diff
@@ -82,7 +85,7 @@ void __attribute__((section("code_l1"))) router_init() {
     my_routing_mode = (EthRouterMode)routing_info->routing_mode;
 }
 
-void __attribute__((section("erisc_l1_code"))) ApplicationHandler(void) {
+void __attribute__((section("erisc_l1_code"))) Application(void) {
```

Comment:
Does this need a corresponding change to `tt_metal/hw/toolchain/erisc-b0-app-sections.ld`?

Reply:
I think it should be ok - the ApplicationHandler function is still the entry (and just calls this function). That seems to be the only section of the .ld file where ApplicationHandler comes up, let me know if I'm missing something here though